Build/Customizations/Extensions.targets

```
@@ -79,4 +79,26 @@
                                                   AssemblyProduct="$(ProductNameForSecurityManager)"
                                                   AssemblyCopyright="$(CopyrightForSecurityManager)" />
   </Target>
+
+  <Target Name="AddNpmDependencyProjectGenerationToBuild" AfterTargets="PrepareBuildTargets">
+    <PropertyGroup>
+      <BuildTargets>
+        $(BuildTargets);
+        GenerateNpmDependencyProjectInOutput;
+      </BuildTargets>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="GenerateNpmDependencyProjectInOutput">
```

Please move this to after-CreateNuGetPackagesWithDebugSymbols was because a) it should be inside the build and before the build-cleanup happens and b) it will then be controlled by the SkipNuGet flag. Right now, we would create it in each build, which isn't nice.

```suggestion
  <Target Name="CreateNpmDependencyPackage" AfterTargets="CreateNuGetPackagesWithDebugSymbols">
```
(better name for "NpmDependencyPackage" ?